x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -697,3 +697,19 @@ c:l| m:i   | m1:i  | c1:l| gender:s
 57 | 25945 | 25945 | 57  | M
 10 | 25324 | 25324 | 10  | null
 ;
+
+
+twoCountStarInStats#[skip:-8.11.99, reason:planning bug fixed in v8.12]
```

Comment:
Add the queries discussed during the reviews, as well.
And, also, some similar queries touching real data, not only `row`s.